roles/calibre-web/templates/calibre-web.service.j2

```diff
@@ -0,0 +1,11 @@
+Description=Calibre-Web
+
+[Service]
+Type=simple
+User={{ calibreweb_user }}
+EnvironmentFile={{ calibreweb_home }}/cps.conf
+ExecStart=/usr/bin/python {{ calibreweb_exec_path }} $ARG1 $ARG2
```

Comment:
@jvonau got this to work by reversing the order of Lines 7 and 8, and tightening them up a bit as follows:
```
WorkingDirectory={{ calibreweb_path }}
ExecStart={{ calibreweb_exec_path }} $ARG1 $ARG2
```
Of course the value of $ARG2 is also critical, as for now it's seems to be pointing to an unworkable `ARG2 = /library/calibre-web/metadata.db` (as specified by [cps.conf](https://github.com/iiab/iiab/pull/999/files#diff-b12619c8160a4ec4a56339e11d6e4b4c)).

Can we go with a tried+true /library/calibre/metadata.db like the one [that IIAB already [creates](https://github.com/iiab/iiab/blob/master/roles/calibre/tasks/create-db.yml) ?